vars/test.yml

```diff
@@ -297,8 +297,8 @@ windows_has_inbox_driver: true
 # 'vmtools_url_path' takes precedence over 'vmtools_iso_path'.
 #
 vmtools_esxi_bundled: false
-vmtools_url_path: "https://packages.vmware.com/tools/releases/12.1.0/windows/VMware-tools-windows-12.1.0-20219665.iso"
-# vmtools_iso_path: "[datastore1] VMware/VMTools/12.1.0/VMware-tools-windows-12.1.0-20219665.iso"
+vmtools_url_path: "https://packages.vmware.com/tools/releases/12.1.5/windows/VMware-tools-windows-12.1.5-20735119.iso"
+# vmtools_iso_path: "[datastore1] VMware/VMTools/12.1.0/VMware-tools-windows-12.1.5-20735119.iso"
```

Comment:
`12.1.0` should also be changed to `12.1.5`. And please commit your changes with command "git commit -s" to sign off your code.